src/app/models/test-type.model.ts

```diff
@@ -15,4 +17,15 @@ export interface TestType {
   prohibitionIssued: boolean;
 
   testResult: string;
+
+  emissionStandard: Emissions.EmissionStandard;
+  smokeTestKLimitApplied: string;
+  fuelType: Emissions.FuelType;
+  modType: {
+    code: Emissions.ModTypeCode;
+    description: Emissions.ModeTypeDescription;
+  } | null;
```

Comment:
Should find out here if `modType` is required but nullable, optional or both. Also, having a separate interface for `ModType` would be clearer here and help us in the future.